src/test/resources/pip/example-requirements.txt

```diff
@@ -0,0 +1,31 @@
+# From: https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
```

Comment:
Are these test files used in a test somewhere?

Reply:
Writing a test for the pip inspector turned out to be outside of the scope of the ticket, so not yet. If you'd prefer I remove this file until a test exists for it I can do that.

Reply:
Currently its purpose is for manual testing.